### DISCUSSION TITLE
How to reload shared data

### DISCUSSION THREAD

**Discussion by infinitetrinity** [inertiajs/inertia#1428] [Help]
This should be straightforward but I can't find any details on how can I reload the shared data?
My shared data "user" is null before login and should be the auth user after login but the data will only be filled after a fresh reload.
So how can I reload the shared data without a full fresh reload?

Thank you

**Comment by G33RY** [comment]
Hi, I think you should have a look at [partial-reloads](https://inertiajs.com/). 

You can use `router.reload({ only: ['user'] })`.

This only reloads data with the given keys.
